getting-started/README.adoc

```diff
@@ -170,6 +171,60 @@ ifndef::qs[]
 . Verify that your service account was successfully created in the *Streams for Apache Kafka* > *Service Accounts* page.
 endif::[]
 
+[id="proc-setting-service-account-permissions_{context}"]
+== Setting permissions for a service account in a Kafka instance in {product}
+
+{product} uses Access Control Lists (ACLs) provided by Kafka that enable you to manage how other user accounts and service accounts are permitted to interact with the Kafka resources that you create. When you create a service account to connect to a Kafka instance, you must also set the appropriate level of permissions for that new account in the ACL of the instance.
```

Comment:
(1) 
> that enable you to manage how other user accounts and service accounts are permitted to interact with the Kafka resources that you create.

I stumbled over this a bit -- does it still make sense if it's shortened to something like one of the following?

- ... manage how other users and service accounts can interact with ...
- ... manage access to the Kafka resources you create

(2) Can we switch from passive without anthropomorphizing? "Kafka includes Access Control Lists (ACLs) so you can manage ..."
 
(3) Maybe flipping the order of this paragraph and/or reconsidering some of the content might help address our often-lengthy intros and create a closer connection with what the user just did? 

- The service account you just created can only describe/view your Kafka instance and any other resources  (maybe too technical to say "describe") 
- You need to use the Access Control Lists (ACLs) Kafka provides to give the service account permission to publish, consume, etc. 
- Maybe this is where we send them off to the more detailed info, esp. re: user accounts, etc.? I will defer to you, the SMEs, and QE here. :D

Reply:
(1) Yeah, I started much more concisely, but then wondered which accounts exactly, so I said "user accounts and service acounts". Then by "can manage" to we mean are able to or are permitted to. So every time I streamlined it, I felt that it introduced some doubt, and since this concept of acess mgmt here is so prone to ambiguities, I opted for the lengthier but clearer explanation to try to avoid the doubt.

(2) I'd have to think on that. At first, I don't know that it feels right.

(3) Great idea here. I had kind of retained the structure similar to the other proc in access mgmt doc, but you're right because out of the context of the access mgmt, the focus should be the action as it falls in this sequence.

I've changed the first paragraph to this. Better?

>After you create a service account to connect to a Kafka instance, you must also set the appropriate level of access for that new account in the Access Control List (ACL) of the Kafka instance. {product} uses ACLs provided by Kafka that enable you to manage how other user accounts and service accounts are permitted to interact with the Kafka resources that you create.